docs/pages/blog/turbo-0-4-0.mdx

```diff
@@ -88,23 +88,23 @@ RUN npm i -g turbo
 
 # Prune the workspace for the `frontend` app
 FROM base as pruner
-COPY .git ./.git
+WORKDIR /app
 COPY . .
-RUN turbo prune --scope=frontend
+RUN turbo prune --scope=frontend --docker
 
 # Add pruned lockfile and package.json's of the pruned subworkspace
-FROM pruner AS installer
-COPY --from=builder /app/out/json/ .
-COPY --from=builder /app/out/yarn.lock ./yarn.lock
+FROM base AS installer
```

Comment:
I believe we'll need: 
`WORKDIR /app` 

Otherwise we're copying everything into the root.

Reply:
Nice catch!